rust/tools/authorization-logic/src/signing/tink_interface.rs

```diff
@@ -0,0 +1,91 @@
+use crate::ast::*;
+use std::fs::File;
+use tink_core::{keyset, TinkError};
+use std::io::{Read,Write, Error};
+
+// The specific choice of how serialization is done is encapsulated because it
+// is likely to change. Binaries are brittle way of serializing objects, but
+// this was reasonable to get running and is easy to change later.
+fn serialize_claim(claim: &Vec<AstAssertion>)-> Vec<u8> {
+    bincode::serialize(&claim).unwrap()
+}
+
+fn deserialize_claim(claim_bin: Vec<u8>)-> Vec<AstAssertion> {
+    bincode::deserialize(&claim_bin[..]).unwrap()
+}
+
+pub fn serialize_to_file(claim: &Vec<AstAssertion>,
+                         filename: &String) -> Result<(), Error> {
+    let mut file = File::create(filename)?;
+    file.write_all(&serialize_claim(claim))?;
+    Ok(())
+}
+
+pub fn deserialize_from_file(filename: &String)
+    -> Result<Vec<AstAssertion>, Error> {
+    let mut file = File::open(filename)?;
+    let mut contents = Vec::new();
+    file.read_to_end(&mut contents)?;
+    Ok(deserialize_claim(contents))
+}
+
+// At present, this function is only used in tests, so a warning will be given
+// that this is not used. In the future, a script for generating keys might
+// also use this outside of tests.
+pub fn store_new_keypair_cleartext(pub_key_file: &String,
+        priv_key_file: &String) {
+    tink_signature::init();
+
+    // create new signing keypair
```

Comment:
nit: Comments should be full sentences starting with a capital and ending with a full-stop.

Reply:
fixed thanks